src/WebUI/Components/ServicesComponent.tsx

```diff
@@ -176,7 +178,7 @@ export class ServicesComponent extends BaseComponent<IServicesComponentPropertie
         let textToRender: string = "";
         switch (column.key) {
             case packageKey:
-                return  ListComponent.renderTwoLineColumn(service.package, service.pipeline, colDataClassName, "primary-text", "secondary-text");
+                return ServicesComponent._getServiceStatus(service, colDataClassName);

```

Comment:
_getServiceStatusWithName() is this a bettername?

Reply:
Done. Changed the method name